OpenRA.Mods.Common/Traits/CustomSelectionSize.cs

```diff
@@ -31,5 +31,7 @@ public int2 SelectionSize(Actor self)
 		{
 			return new int2(info.CustomBounds[0], info.CustomBounds[1]);
 		}
+
+		public int2 SelectionOffset(Actor self) { return int2.Zero; }
```

Comment:
We could use this opportunity to make `CustomSelectionSize` more flexible and useful, by returning `new int2(info.CustomBounds[2], info.CustomBounds[3])` here if `info.CustomBounds.Length > 2`.